src/main/app/src/app/gebruikersvoorkeuren/zoekopdracht/zoekopdracht.component.ts

```diff
@@ -42,6 +43,41 @@ export class ZoekopdrachtComponent implements OnInit, OnDestroy {
         });
     }
 
+    private heeftZoekParameters(): boolean {
+        const parameters: any = this.zoekParameters;
```

Comment:
zoekparameters = ZoekParameters | ListParameters. Ik zou of dat type gebruiken of any weglaten